app/Services/ContainerStatusAggregator.php

```
@@ -0,0 +1,253 @@
+        return $this->resolveStatus(
+            $hasRunning,
+            $hasRestarting,
+            $hasUnhealthy,
+            $hasUnknown,
+            $hasExited,
+            $hasStarting,
+            $hasPaused,
+            $hasDead,
+            $maxRestartCount
+        );
+    }
```

PHPMD (2.15.0)

The method aggregateFromContainers() has a Cyclomatic Complexity of 16. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)